javascript/example_code/iam/iam_accesskeylastused.js

```diff
@@ -22,8 +22,8 @@ var iam = new AWS.IAM({apiVersion: '2010-05-08'});
 
 iam.getAccessKeyLastUsed({AccessKeyId: 'ACCESS_KEY_ID'}, function(err, data) {
   if (err) {
-    console.log("Error", err);
+    throw err;
```

Comment:
I changed all the error console logs to throws, since these examples are written as command line tools. This will cause the script to properly report an error, vs exiting with a successful exit code.